### PR TITLE
REST tests for PublicAPI and export/import

### PR DESCRIPTION
**Pull request by mkralik3**
//test: `@publicapi`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mkralik3** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mkralik3** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mkralik3** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mkralik3** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mkralik3** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: rest-tests/src/test/java/io/syndesis/qe/CommonHooks.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package io.syndesis.qe;
+
+import cucumber.api.java.After;
+import io.syndesis.qe.endpoints.TestSupport;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class CommonHooks {
```
Comment (avano) [inline rest-tests/src/test/java/io/syndesis/qe/CommonHooks.java]: there is already a class defined for the hooks - `io.syndesis.qe.RestTestHooks`

File: utilities/src/main/java/io/syndesis/qe/bdd/validation/CommonValidationSteps.java [hunk 2]
```diff
@@ -86,6 +88,17 @@ public void verifyNoIntegrationBuildRunning(String integrationName) {
         log.info("There is no builds with name {} running", sanitizedName);
     }
 
+    @Then("^verify that integration with name (\\w+) exist$")
+    public void integrationExist(String integrationName) {
+        assertThat(integrationsEndpoint.getIntegrationId(integrationName).isPresent()).isTrue();
```
Comment (avano) [inline utilities/src/main/java/io/syndesis/qe/bdd/validation/CommonValidationSteps.java]: assertThat(optional).isPresent()

File: rest-tests/src/test/java/io/syndesis/qe/CommonHooks.java (hunk 1 above)
Comment (mkralik3) [reply rest-tests/src/test/java/io/syndesis/qe/CommonHooks.java]: I didn't notice that. thx

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mkralik3** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: rest-tests/src/test/resources/features/publicapi/integrations.feature [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+      | tag1 | tag12 | importedTag |
+    And check that integration integration2 contains exactly tags
+      | tag3 | tag12 | importedTag |
+    And check that integration integrationWithoutTags contains exactly tags
+      | anotherTag1 | anotherTag2 |
+
+  # GET ​/public​/integrations​/{env}​/export.zip?all=true
+  # POST ​/public​/integrations
+  @export-import-all-integrations
+  Scenario: Export and import integrations according to tag
```
Comment (avano) [inline rest-tests/src/test/resources/features/publicapi/integrations.feature]: pls make this scenario name unique

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mkralik3** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Pull request merged**